goalgorilla/open_social
# Issue #2896168 by maikelkoopman: replace autogrow with autosize library

### How to test:
- create a new post with a few paragraphs
- edit the post, the textarea should be as tall as the text.
- See that autogrow behaviour still works as expected on comment fields
